### ISSUE TITLE
Wizard-Lich is incredibly too strong.

### ISSUE THREAD

**Issue by Kokojos** [HippieStationArchive/HippieStation13-1#2752]
It's a perceived BALANCE issue please discuss.
- Infinite revives.
- Most enchanted items are not super obvious (pens)
- The directions it gives when you kill a lich is super hard to follow (towards the west, woa so precise)
- Ghost knowledge while dead.

**Comment by ghost** [comment]
Agreed, but then it's nothing new.

Wizard is completely fucked in terms of balance. Needs a wholesale rework.

**Comment by GenericGoose** [comment]
wizard is fucking broken dude no surprise
make a fancy list with milestones on all magicman's overpoweredness

**Comment by ghost** [comment]
NOT A BUG
